packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -387,19 +384,11 @@ const findDefaultExportModuleNode = (
   foundNode: t.Node | undefined;
   foundParentPath: NodePath | undefined;
 } => {
-  let foundNode: t.Node | undefined = undefined;
-  let foundParentPath: NodePath | undefined = undefined;
-
-  traverse(ast, {
-    ExportDefaultDeclaration(path) {
-      foundParentPath = path as NodePath;
-      foundNode = path.node.declaration as t.Node;
-    },
-  });
+  const result = getDefaultExport(ast);
```

Comment:
Since this function `findDefaultExportModuleNode ` and `findNamedExportModuleNode` don't do anything anymore, can you remove them and just inline the logic at the call sites?